Makefile

```diff
@@ -473,13 +476,15 @@ test-e2e: test-e2e-setup install-ginkgo
 	--ginkgo.timeout=2h
 
 .PHONY: test-e2e-cleanup
-test-e2e-cleanup:
-	oc delete volumesnapshotcontent --all
-	oc delete volumesnapshotclass oadp-example-snapclass --ignore-not-found=true
-	oc delete backup -n $(OADP_TEST_NAMESPACE) --all
-	oc delete backuprepository -n $(OADP_TEST_NAMESPACE) --all
-	oc delete downloadrequest -n $(OADP_TEST_NAMESPACE) --all
-	oc delete podvolumerestore -n $(OADP_TEST_NAMESPACE) --all
-	oc delete restore -n $(OADP_TEST_NAMESPACE) --all --wait=false
-	for restore_name in $(shell oc get restore -n $(OADP_TEST_NAMESPACE) -o name);do oc patch "$$restore_name" -n $(OADP_TEST_NAMESPACE) -p '{"metadata":{"finalizers":null}}' --type=merge;done
+test-e2e-cleanup: login-required
+	$(OC_CLI) delete volumesnapshotcontent --all
+	$(OC_CLI) delete volumesnapshotclass oadp-example-snapclass --ignore-not-found=true
+	$(OC_CLI) delete backup -n $(OADP_TEST_NAMESPACE) --all
```

Comment:
If I delete backup, it deletes dataupload. Should I change order or just add the ones that delete another entities here?

Reply:
I would add `--wait` and move all data that velero typically cleans up on backup deletion last.